spec/mongoid/query_cache_spec.rb

```diff
@@ -543,7 +557,7 @@
       Mongoid::QueryCache.enabled = true
       10.times { Band.create! }
 
-      Band.batch_size(4).all.any?
+      Band.batch_size(4).all
```

Comment:
This changes from executing the query to not executing, no?

Reply:
Yup good catch. Meant to change it to `to_a`.

Reply:
Block methods like `any?` currently cause session leaks as described by [RUBY-2391](https://jira.mongodb.org/browse/RUBY-2391).